src/components/one-time-donation/components/AnonimusMenu.tsx

```diff
@@ -0,0 +1,97 @@
+import { Collapse, Grid, List, Typography } from '@mui/material'
+import * as React from 'react'
+import { createStyles, makeStyles } from '@mui/styles'
+import FormTextField from 'components/common/form/FormTextField'
+import CheckboxField from './FormCheckField'
+import { useField } from 'formik'
+import { useTranslation } from 'next-i18next'
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    message: {
+      maxWidth: '540px',
+      height: '70px',
+      background: '#FFFFFF',
+      borderRadius: '60px',
+      textAlign: 'center',
+    },
+  }),
+)
+
+export default function AnonimusMenu() {
+  const classes = useStyles()
+  const [field] = useField('anonymousDonation')
+  const { t } = useTranslation('one-time-donation')
+
+  return (
+    <>
+      <CheckboxField label={t('anonimus-menu.checkbox-label') as string} name="anonymousDonation" />
+      <Collapse in={field.value} timeout="auto" unmountOnExit>
+        <List component="div" disablePadding>
+          <Typography>{t('anonimus-menu.info-start')}</Typography>
+          <Grid my={'35px'}>
```

Comment:
For all margins you should use `theme.spacing(amount)`

Reply:
An easy way to use the theme in `sx` prop is as follows:


```tsx
<Grid sx={theme => ({ my: theme.spacing(5) })}>
```